src/getMigrateWidgets.ts

```diff
@@ -44,22 +48,53 @@ export const getMigrateWidgets =
       contentServer.children?.ui.children?.widgets.children?.structure!;
     const filesAncestry = _getFilesAncestry(widgetsStructure);
 
+    const migrateWidget = produce(callback);
```

Comment:
Just to make sure I understand, `produce` here returns a function that executes the callback when called, right?

Reply:
Yes, it's the curried form of `produce`.